mail-filter/mimedefang/files/mimedefang.conf

```diff
@@ -0,0 +1,131 @@
+# Additional options for MIMEDefang.
+# read man pages mimedefang(8) for additional informations.
+
+# Locale should be set to "C" for generating valid date headers
```

Comment:
That sounds like a bug in the program. Wouldn't it be better to patch it there?

Reply:
https://lists.roaringpenguin.com/pipermail/mimedefang/2019-June/038366.html